src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html

```diff
@@ -0,0 +1,73 @@
+<div class="research">
+  <div class="search-input"><input type="text" placeholder="Search" (keyup)="applyFilter($event.target.value)" /></div>
+  <div class="icon-style"><i class="fa fa-search" aria-hidden="true"></i></div>
+</div>
+<div class="btn-group">
+  <div class="select-group">
+    <img [src]="ubsAdminTableIcons.arrowDown" alt="arrow down" />
+    <select class="display-counter" (change)="selectPageSize($event.target.value)">
+      <option [value]="size" *ngFor="let size of pageSizeOptions">{{ size }}</option>
+    </select>
+  </div>
+  <button type="button" class="btn btn-default" class="view-table-btn">
+    <mat-icon>create</mat-icon>
+    Додати сертифікат
+  </button>
+  <button type="button" class="btn btn-default" class="export-excel-btn">Експортувати в Excel</button>
```

Comment:
Can we move the text to .json files and use the translation here?

Reply:
done